content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -262,6 +262,17 @@ Secret somewhere that your terminal / computer screen could be seen by an onlook
 When you delete a ServiceAccount that has an associated Secret, the Kubernetes
 control plane automatically cleans up the long-lived token from that Secret.
 
+{{< note >}}
+If you view the ServiceAccount using:
+
+` kubectl get serviceaccount build-robot -o yaml`
+
+You can't see the `build-robot-secret` Secret in the ServiceAccount API objects
+[`.secrets`](/docs/reference/kubernetes-api/authentication-resources/service-account-v1/) field
+because manually created Secret API objects do not appear in the ServiceAccount API object's
+`.secrets` field. That field is only populated with auto-generated Secrets.
```

Comment:
This can be simplified, right? Talking straight to the point?

```suggestion
because that field is only populated with auto-generated Secrets.
```

Reply:
yeah, it seems good. I will modify it.

Thanks!

Reply:
Addressed the above comment.